testing/tests/DevExpress.ui.widgets.form/form.API.option_formData.tests.js

```diff
@@ -329,3 +329,165 @@ QUnit.test('Reset editor\'s value when set formData: {dataField1: a}', function(
+                const oldFormData = { a: true };
+                if(oldBoolValue !== 'no member') {
+                    oldFormData['b'] = oldBoolValue;
+                }
+
+                const form = $('#form').dxForm({
+                    formData: oldFormData,
+                    items: [
+                        { dataField: 'a', editorType: 'dxCheckBox' },
+                        { dataField: 'b', editorType: 'dxCheckBox' }
+                    ],
+                }).dxForm('instance');
+
+                const expectedOldValue = (oldBoolValue === undefined || oldBoolValue === 'no member') ? false : oldBoolValue;
+                checkCheckboxEditor(form.getEditor('a'), true);
+                checkCheckboxEditor(form.getEditor('b'), expectedOldValue);
+
+                const newFormData = {};
+                if(newBoolValue !== 'no member') {
+                    newFormData['b'] = newBoolValue;
+                }
+
+                form.updateData(newFormData);
+                const expectedNewValue = newBoolValue === 'no member' ? expectedOldValue : newBoolValue;
+                checkCheckboxEditor(form.getEditor('a'), true);
+                checkCheckboxEditor(form.getEditor('b'), expectedNewValue);
+            });
+
+            QUnit.test(`FormData = { innerObject: { a: true, b:  ${oldBoolValue} }} -> updateFormData({ innerObject.b = ${newBoolValue })`, function() {
+                const oldFormData = { innerObject: { a: true } };
+                if(oldBoolValue !== 'no member') {
+                    oldFormData.innerObject['b'] = oldBoolValue;
+                }
+
+                const form = $('#form').dxForm({
+                    formData: oldFormData,
+                    items: [
+                        { dataField: 'innerObject.a', editorType: 'dxCheckBox' },
+                        { dataField: 'innerObject.b', editorType: 'dxCheckBox' }
+                    ],
+                }).dxForm('instance');
+
+                const expectedOldValue = (oldBoolValue === undefined || oldBoolValue === 'no member') ? false : oldBoolValue;
+                checkCheckboxEditor(form.getEditor('innerObject.a'), true);
+                checkCheckboxEditor(form.getEditor('innerObject.b'), expectedOldValue);
+
+                const newFormData = { innerObject: { a: true } };
+                if(newBoolValue !== 'no member') {
+                    newFormData.innerObject['b'] = newBoolValue;
+                }
+
+                form.updateData(newFormData);
+                const expectedNewValue = newBoolValue === 'no member' ? expectedOldValue : newBoolValue;
+                checkCheckboxEditor(form.getEditor('innerObject.a'), true);
+                checkCheckboxEditor(form.getEditor('innerObject.b'), expectedNewValue);
+            });
+        });
+
+        QUnit.test(`FormData = { a: true, b:  ${oldBoolValue}} -> option('formData', {})`, function() {
```

Comment:
This config seems to be covered by

    [true, false, undefined, null, 'no member'].forEach(oldBoolValue => {
        ['no member'].forEach(newBoolValue => {

Remove?